### DISCUSSION TITLE
Hibernate Query

### DISCUSSION THREAD

**Discussion by Vashirr01** [mylinuxforwork/dotfiles#365] [General]
When i hibernate, all data and windows open are lost. How do i make the data and windows stay open like in KDE plasma?

**Comment by mylinuxforwork** [comment]
Very good question. Currently I have no clue. Will start soon more tests on Laptops and multi monitor setups.

**Reply by Vashirr01** [reply]
i understand. but can you do this generally in hyprland?
i'm loving how everything's setup in ml4w btw. very seamless
